CanvasPlusPlayground/Features/Courses/CourseListView.swift

```diff
@@ -216,6 +219,18 @@ private struct CourseListCell: View {
                     course.isFavorite = !wrappedCourseIsFavorite
                 }
             }
+            
+            Button("Add Course Nickname", systemImage: "character.cursor.ibeam") {
+                withAnimation {
+                    showRenameTextField = true
+                }
+            }
+            
+            Button("Remove Course Nickname") {
+                withAnimation {
+                    course.nickname = nil
+                }
+            }
```

Comment:
I think a better UX might be to just have a "rename" button, and if the textfield text is empty in the alert, we set the nickname to nil. In this implementation, it seems like I'm able to have an empty nickname.

Reply:
addressed!